### PR TITLE
Cleanup dockerfile and add coral support

### PR DESCRIPTION
**Pull request by ethanfrey**
* Remove ledger support from built artifacts (longer buildtime, deps, not useful in docker)
* Add `coral`/`corald` binaries

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by ethanfrey** [comment]
I would like multiple dockerfiles sometime in the future, but that makes the build time huge 

Let's revisit this later, but I would do this for coral and gaiaflex

File: Dockerfile [hunk 1]
```diff
@@ -24,16 +24,18 @@ RUN export GO_WASM_DIR=$(go list -f "{{ .Dir }}" -m github.com/CosmWasm/go-cosmw
 FROM cosmwasm/go-ext-builder:0.8.2-alpine AS go-builder
 
 RUN apk add git
-# without this, build with LEDGER_ENABLED=false
-RUN apk add libusb-dev linux-headers
+# NOTE: add these to run with LEDGER_ENABLED=true
+# RUN apk add libusb-dev linux-headers
```
Comment (webmaster128) [inline Dockerfile:28]: There is another block like this a few lines above (line 6-7). Should we do the same there as well?

Comment (ethanfrey) [reply Dockerfile:28]: Good call. I can remove that one entirely.
I guess I'll throw that in an existing branch - #229 to avoid a micro-pr

Comment (ethanfrey) [reply Dockerfile:28]: https://github.com/CosmWasm/wasmd/pull/229/commits/08aef4a650afb94b4a80f64d80d185087629348e

**Pull request merged**